lib/phonebook/providers/association_member_list_provider.dart

```diff
@@ -16,10 +18,13 @@ class AssociationMemberListNotifier extends ListNotifier<CompleteMember> {
   }
 
   Future<AsyncValue<List<CompleteMember>>> loadMembers(
-      String associationId, String year) async {
+      String associationId, String year, ref) async {
```

Comment:
Why are you passing ref here ?

Reply:
[fixed here](https://github.com/aeecleclair/Titan/pull/41/commits/415e717cca8877c95e8b93df45990985799c60b6)